department-of-veterans-affairs/va.gov-cms
# VACMS-4017: CC-Policies-Proofing

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/4017

## Testing done


## Screenshots


## QA steps

- Login to preview environment
- Navigate here: `/node/add/vamc_system_policies_page`
- [ ] Confirm tooltip exists on both the edit and preview screen and is able to show the description on hover and on tab

![image](https://user-images.githubusercontent.com/56809719/106213616-12502700-619b-11eb-9ea6-a8323fa68b9e.png)


### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
